.update-copyright.conf

```diff
@@ -0,0 +1,17 @@
+[project]
+name: lesson-template
+vcs: Git
+
+[files]
+authors: yes
+files: no
+
+[copyright]
+short: {project} comes with ABSOLUTELY NO WARRANTY and is licensed under the MIT and CC BY 4.0 licenses.
+long: This file is part of {project}.
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
I prefer "See LICENSE.md for copyright and license information." -- 
single unambiguous source of the information.